pyproject.toml

```diff
@@ -62,8 +62,9 @@ pre-commit = "2.15.0"
 beautifulsoup4 = ">=4.8,<4.10.0"
 coverage = "6.1.1"
 pymdown-extensions = "9.0"
-poetry = "1.1.13"
 syrupy = "4.0.1"
+poetry = ">1.0,<=1.4.1"
```

Comment:
I think we should be more specific about this, e.g. `>=1.4,<1.5`

Reply:
Any lower bound higher than 1.3 errors when running poetry lock. I end up with `So, because tbxforms depends on both poetry (>1.3,<1.5) and tox (>=4.4,<4.5), version solving failed.`

Reply:
@zerolab can you see any issue with removing `poetry` from `[tool.poetry.dev-dependencies]` here? It seems odd to me that it should be included.